handler/database/sql/sql.go

```diff
@@ -1,17 +1,41 @@
-package handler
+package sql
 
 import (
 	"encoding/json"
+	"fmt"
 
+	"github.com/DATA-DOG/godog"
 	"github.com/DATA-DOG/godog/gherkin"
 	"github.com/tomatool/tomato/conv"
 	"github.com/tomatool/tomato/errors"
+	"github.com/tomatool/tomato/resource"
 )
 
+type DatabaseSQL interface {
+	resource.Resource
+
+	Select(tableName string, condition map[string]string) ([]map[string]string, error)
+	Insert(tableName string, rows []map[string]string) error
+	Delete(tableName string, condition map[string]string) (int, error)
+}
+
+type Handler struct {
+	r map[string]DatabaseSQL
+}
+
+func New(r map[string]DatabaseSQL) *Handler {
```

Comment:
I don't think this New() is necessary if it just sets the Handler map field

Maybe something like:

type Handlers map[string]Database?